iroh-net-report/src/lib.rs

```diff
@@ -344,17 +375,67 @@ impl Options {
     }
 }
 
+/// Options for running probes (in browsers).
+///
+/// Only HTTPS probes are supported in browsers.
+/// These are run by default.
+#[cfg(wasm_browser)]
+#[derive(Debug, Clone)]
+pub struct Options {
+    /// Enable https probes
+    ///
+    /// On by default
+    https: bool,
+}
+
+#[cfg(wasm_browser)]
+impl Default for Options {
+    fn default() -> Self {
+        Self { https: true }
+    }
+}
+
+#[cfg(wasm_browser)]
+impl Options {
```

Comment:
maybe have a submodule for `options` one for wasm and one for the browser?

```rust

pub use options::Options;

#[cfg(wasm_browser)]
mod options {
  pub struct Options
}

#[cfg(not(wasm_browser))]
mod options {
  pub struct Options
}
```rust